src/utils/mev.ts

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios'
+import { ethers } from 'ethers'
+const logger = require('../services/logger')
+
+const bevUrl = 'https://bev-relay.kolibr.io'
+
+export function isEthSendMethod(rawData) {
+  if (rawData instanceof Array) {
```

Comment:
why this is false on an array? there's a possibility that one or more of the methods within the array is `eth_sendRawTransaction`

Reply:
It would be pretty difficult to split off an array and inject the correct response back into the array of the transactions in the order it came through that the consumer would expect.